posthog/tasks/status_report.py

```diff
@@ -63,31 +69,57 @@ def status_report(*, dry_run: bool = False) -> Dict[str, Any]:
 
     for team in Team.objects.exclude(organization__for_internal_metrics=True):
         try:
+            params = (team.id, report["period"]["start_inclusive"], report["period"]["end_inclusive"])
             team_report: Dict[str, Any] = {}
-            events_considered_total = Event.objects.filter(team_id=team.id)
-            instance_usage_summary["events_count_total"] += events_considered_total.count()
-            events_considered_new_in_period = events_considered_total.filter(
-                timestamp__gte=period_start, timestamp__lte=period_end,
-            )
+            if is_clickhouse_enabled():
+                # pull events stats from clickhouse
+                from ee.clickhouse.models.event import (
+                    get_event_count_for_team,
+                    get_event_count_for_team_and_period,
+                    get_events_count_for_team_by_client_lib,
+                    get_events_count_for_team_by_event_type,
+                )
+
+                team_event_count = get_event_count_for_team(team.id)
+                instance_usage_summary["events_count_total"] += team_event_count
+                team_report["events_count_total"] = team_event_count
+                team_events_in_period_count = get_event_count_for_team_and_period(team.id, period_start, period_end)
+                team_report["events_count_new_in_period"] = team_events_in_period_count
+                instance_usage_summary["events_count_new_in_period"] += team_report["events_count_new_in_period"]
```

Comment:
```suggestion
                instance_usage_summary["events_count_new_in_period"] += team_events_in_period_count
```
nit for readability/consistency